src/ts/components/subtitleoverlay.ts

```diff
@@ -474,22 +487,20 @@ export class SubtitleRegionContainerManager {
       }
     }
 
-    this.subtitleRegionContainers[regionName].addLabel(label);
+    this.subtitleRegionContainers[regionName].addLabel(label, overlaySize);
   }
 
   /**
    * Removes a subtitle label from a container.
    */
   removeLabel(label: SubtitleLabel): void {
-    const region = label.region || 'default';
-    for (const regionName in this.subtitleRegionContainers) {
```

Comment:
do you know why this loop was there before?

Reply:
Not sure but @hanneslinder said that is is fine for us to remove them as they do not affect anything